### PR TITLE
Add tier 4 area data

### PR DESCRIPTION
**Pull request by JonathanHallam**
Update to the local restriction data with tier 4 areas.

### CODE REVIEW

File: config/local_restrictions.yml [hunk 1]
```diff
@@ -1557,165 +1563,201 @@ E07000094:
...
+      start_time: "00:01"
 E07000114:
   name: Thanet District Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+      - alert_level: 4
+        start_date: 2020-12-21
+        start_time: "00:01"
```
Comment (richardTowers) [inline config/local_restrictions.yml:1742]: This is indented one level too far.

```suggestion
    - alert_level: 4
      start_date: 2020-12-21
      start_time: "00:01"
```

File: config/local_restrictions.yml [hunk 2]
```diff
@@ -179,105 +179,111 @@ E06000030:
...
   name: Peterborough City Council
   restrictions:
-    - alert_level: 2
-      start_date: 2020-12-02
-      start_time: "00:01"
     - alert_level: 3
       start_date: 2020-12-19
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
```
Comment (leenagupte) [inline config/local_restrictions.yml:186]: I think this needs to be `2020-12-20`

File: config/local_restrictions.yml (hunk 1 above)
Comment (JonathanHallam) [reply config/local_restrictions.yml]: good spot, thanks

File: config/local_restrictions.yml (hunk 2 above)
Comment (JonathanHallam) [reply config/local_restrictions.yml]: 🤦 I copied it from an earlier notes file after my editor crashed. Replaced them all now

**Comment by richardTowers** [comment]
Some `jq` sneaky tricks that I'm using to review this:

```
ruby -rjson -ryaml -e 'puts YAML.load_file("config/local_restrictions.yml").to_json' |
  jq -r '.[] | select(.restrictions[].alert_level == 4) | .name'
```

Returns a list of 98 LTLAs which are moving into tier 4. I'm just matching them up with the spreadsheet.

**Review by richardTowers — APPROVED** [review]
I've checked that:

* The LTLAs moving to tier 4 are the same as in the spreadsheet that was signed off
* Every tier 4 alert level has the same configuration
* The tiers being removed are all in the past

**Review by richardTowers — APPROVED** [review]
Also tested manually for a few postcodes.

**Review by leenagupte — APPROVED** [review]
🥇

**Pull request merged**